examples/distributed-key-value-store/src/main.rs

```diff
@@ -181,7 +182,7 @@ fn handle_input_line(kademlia: &mut kad::Behaviour<MemoryStore>, line: String) {
             };
             let value = {
                 match args.next() {
-                    Some(value) => value.as_bytes().to_vec(),
+                    Some(value) => Bytes::from(value.as_bytes().to_owned()),
```

Comment:
This is a double allocation, use `copy_from_slice` instead.

Reply:
Ah nice one. I was wondering what the correct way was here 💪